### DISCUSSION TITLE
Lilygo T-Dongle-S3 w/Display

### DISCUSSION THREAD

**Discussion by PCgoDK** [BIPES/BIPES#237] [Q&A] [answered]
ESP32-S3 support? And especially Lilygo T-Dongle-S3 w/Display
Is it possible to use display ST7735 in Bipes?
It seems like GPIO38 are needed but not available in Bipes:
https://www.lilygo.cc/products/t-dongle-s3

**Comment by sk8board** [comment]
BIPES is an application that has more support for Raspberry Pi Pico, but does not support the S3 or the ST7735 display. 

For support of ESP32 and other displays, I recommend you try https://quirkycort.github.io/IoTy/public/editor.html

**Reply by PCgoDK** [reply, marked as answer]
Thank you for this link. It looks promising. 
I just need GPIO 38 to turn on backlight on TFT to have perfect support. But you guided me well towards alternatives..
Thank you for contributing

**Comment by jetannenbaum** [comment]
The ST7735 library is available on github and I could add it for the Pico.  Unfortunately, I don't support the ESP32-S3 since I don't have one.  Would you like me to add support for the display to the Pico?

**Reply by PCgoDK** [reply]
No not needed in my case. 
I have chosen to work with this very compact ESP32 device that include all the hardware needed for the project.
Thank you for asking :)
